tools/dockerfile/grpc_artifact_python_manylinux2010_x64/Dockerfile

```diff
@@ -13,13 +13,14 @@
 # limitations under the License.
 
 # Docker file for building gRPC manylinux Python artifacts.
-# Updated: 2020-10-08
 
-FROM quay.io/pypa/manylinux2010_x86_64
+# python2.7 was removed from the manylinux2010 image.
+# Use the latest version that still has python2.7
+# TODO(jtattermusch): Stop building python2.7 wheels.
```

Comment:
Stop releasing 2.7 could be considered as a minor change.

Reply:
Let's look into stopping 2.7 releases as a separate PR (it's a user visible change so it makes sense to have a clear defined PR for it and there's potential for needing to roll it back for unexpected reasons).